es6/tests/e2e/table.js

```diff
@@ -18,139 +20,147 @@ describe("Table", function () {
 				},
 			],
 		};
-		shouldBeSame({
-			doc: createDocV4("loop-valid.docx", { paragraphLoop: true }).render(tags),
-			expectedName: "expected-loop-valid.docx",
-		});
+		this.expectedName= "expected-loop-valid.docx";
+	
+		return this.renderV4(this.data);
 	});
 
 	it("should work with tables", function () {
-		const tags = {
+		this.data = {
 			clients: [
 				{ first_name: "John", last_name: "Doe", phone: "+33647874513" },
 				{ first_name: "Jane", last_name: "Doe", phone: "+33454540124" },
 				{ first_name: "Phil", last_name: "Kiel", phone: "+44578451245" },
 				{ first_name: "Dave", last_name: "Sto", phone: "+44548787984" },
 			],
 		};
-		const doc = createDoc("tag-intelligent-loop-table.docx")
-			.setData(tags)
-			.render();
+		this.name = "tag-intelligent-loop-table.docx";
+		this.expectedName = "expected-tag-intelligent-loop-table.docx";
 		const expectedText =
```

Comment:
This test doesn't test docxtemplater anymore if you set text = this.data.clients.map......

You have to create a new property called : 

`this.expectedText = "JohnDoe+33647874513JaneDoe+33454540124PhilKiel+44578451245DaveSto+44548787984"`

Then, in the code of this.render(), you have to check if this.expectedText exists, and if does, do the `expect(doc.getFullText()).to.be.equal(this.expectedText);` in the `render` method.